internal/routes/routes.go

```
@@ -8,84 +8,104 @@ import (
 func SetupMovieRoutes(bot *telebot.Bot, container *api.Resolver, app *appCfg.App) {
+	const op = "routes.SetupMovieRoutes"
 	bot.Handle("/sm", middleware.RequireTMDBToken(container.MovieHandler.SearchMovie, app))
 }
 
 func SetupTVRoutes(bot *telebot.Bot, container *api.Resolver, app *appCfg.App) {
+	const op = "routes.SetupTVRoutes"
 	bot.Handle("/stv", middleware.RequireTMDBToken(container.TVHandler.SearchTV, app))
 }
 
 func SetupInfoRoutes(bot *telebot.Bot, container *api.Resolver, app *appCfg.App) {
+	const op = "routes.SetupInfoRoutes"
```

golangci-lint (1.64.8)

const `op` is unused

(unused)